### ISSUE TITLE
Holes by design are filled

### ISSUE THREAD

**Issue by MatousRokosVolue** [Ultimaker/Cura#12091]
### Application Version

5.0

### Platform

Windows 11

### Printer

Snapmaker A350

### Reproduction steps

Object with the hole from the bottom by design gets filled and sliced solid. 
Actual object: 
![image](https://user-images.githubusercontent.com/62334499/166809757-300cd818-02ae-4d74-b6c2-b8369ac136d4.png)

### Actual results

Sliced preview:
![image](https://user-images.githubusercontent.com/62334499/166809880-c3212f3c-461c-4873-ae28-6f9858e7e7f9.png)

The hole is actually visible in preview yet filled anyway:
![image](https://user-images.githubusercontent.com/62334499/166810024-6d82a4f1-2ef7-451e-8121-68f57974268c.png)

### Expected results

Holes should be printed as expected and rendered in sliced preview

### Checklist of files to include

- [ ] Log file
- [X] Project file

### Additional information & file uploads

[S2A_bear-3.zip](https://github.com/Ultimaker/Cura/files/8626523/S2A_bear-3.zip)

**Comment by smartavionics** [comment]
Interestingly, if you turn the model on its side, the hole appears!

![Screenshot 2022-05-04 21 05 16](https://user-images.githubusercontent.com/585618/166817225-44c004db-b10f-48d4-959a-7b98e8ab7b4b.png)

**Comment by GregValiant** [comment]
It's the infamous curse of "Make Overhangs Printable".  Turn that off as it is for extremely specific and not often encountered problems.
![image](https://user-images.githubusercontent.com/64202104/166820088-1e1f9548-b75d-4362-90dd-53ecae9c3492.png)

**Comment by MatousRokosVolue** [comment]
yeah, that's it. Can confirm that "Make Overhangs Printable" setting does it in both 5.0 and 4.13.1. Still relevant as bug report perhaps :)

**Comment by GregValiant** [comment]
It's "expected behavior".  The setting is one I've never had to use but I know it has helped with some problems (not many).
"Remove all Holes" is another one.  New users seem to figure it's a good idea and turn it on.  Then it "Removes all Holes" and they wonder what happened.
If you happen to need either one then they come in handy.  Folks just need to remember to turn them off when finished.

**Comment by MatousRokosVolue** [comment]
"remove holes" I'd understand but "make overhangs printable".... Well it's hard not to print overhangs if you don't have them in the first place :)

**Comment by GregValiant** [comment]
The top of that hole has an overhang / double bevel / chamfer, whatever you want to call it.  I think that's what caused the hole to fill up.  As long as it's sorted out it's all good.
